_includes/install_instructions/shell.html

```diff
@@ -14,6 +14,165 @@ <h3>The Bash Shell</h3>
+                </ul>
+            </li>
+            <li>Follow the <a href="#shell-windows-wsl-install-software">Installing software within WSL 2</a> section below.</li>
+        </ol>
+
+        <h4 id="shell-windows-wsl-powershell">Via Powershell:</h4>
+        <ol>
+            <li>Click on the Windows Start Menu button in the bottom left corner of your screen, or tap the Windows button on your keyboard, and type “powershell” in the search bar.</li>
+            <li>Open Powershell in administrator mode by clicking “Run as Administrator” in the right hand side pane:
+                <ul>
+                    <li>If a pop-up window appears asking if you want to make changes to your device, select “Yes”</li>
+                    <li><strong>If you are unable to run in Administrator mode</strong>, you may have to contact your IT support personnel to ask for access or to install WSL 2 for you</li>
+                </ul>
+            </li>
+
+            <li>At the Powershell prompt, type <code>wsl -–install</code>:
+                <ul>
+                    <li>The installer will download Ubuntu by default, and this may take a few minutes</li>
+                    <li>Once installed, you will be prompted to enter in a new username and password. These will be only set for your Ubuntu environment, and are not related to your Windows operating system.</li>
+                    <li>Upon completion, you will be presented with the bash prompt.</li>
+                </ul>
+            </li>
+            <li>Follow the <a href="#shell-windows-wsl-install-software">Installing software within WSL 2</a> section below.</li>
+        </ol>
+
+        <h4 id="shell-windows-wsl-install-software">Installing software within WSL 2</h4>
+        <p>
+            Once you have installed WSL 2, you will need to install some software within the Ubuntu operating system in order to use it effectively. Make sure you are using Ubuntu by checking you see the bash prompt:
+        </p>
+
+        <p>
+            The first thing to do is to update the package manager's list of available packages. To do this in Ubuntu, we use a tool called <code>apt</code>.
+            Type the following command into the bash prompt:
+            <code>sudo apt update</code>
+        </p>
+        <p>
+            <strong>Note:</strong> You will be prompted to enter your password. This is the password you set when installing Ubuntu, and will not show up on the screen as you type it in.
+            This is a security feature of the terminal, and is normal behaviour.
+        </p>
+        <p>
+            Install updated versions of packages required across all our lessons, by typing:
+            <ul>
+                <li><code>sudo apt-get install git build-essential autotools python3 r-base</code></li>
+            </ul>
+        </p>
```

Comment:
Would you expect `apt update` to take a long time to run here? I genuinely have no idea but if so, I think it's worth including a comment that this is expected.